docs/backend_architecture/tasks.rst

```diff
@@ -6,6 +6,15 @@ Kolibri plugins and Django apps can use the backend tasks system to run time con
 
 The kolibri task system is implemented as a core Django app on ``kolibri.core.tasks``.
 
+Kolibri backend tasks system flow diagram
+-------------------------------------------
+
+The following diagram explains how a task travels from the frontend client to the different parts of the backend task system. It aims to give a high level understanding of the backend tasks system.
+
+You should download the following image to be able to zoom it in your image viewer. You can download by right clicking on following image and select "save image as" option.
+
+.. image:: ./kolibri_task_system.png
+
```

Comment:
If there's an editable source that we could use in the case we need to update the diagram, would be good to add here. See the bottom of this file, as an example: https://raw.githubusercontent.com/learningequality/kolibri/develop/docs/dataflow/index.rst

Reply:
I've added an editable link of the diagram sir.